javascript-forms/form-values.js

```diff
@@ -0,0 +1,18 @@
+var $contactForm = document.querySelector('#contact-form');
+
+$contactForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+
+  var contactInfo = {};
+
+  contactInfo[$contactForm[0].name] = $contactForm[0].value;
+
+  contactInfo[$contactForm[1].name] = $contactForm[1].value;
+
+  contactInfo[$contactForm[2].name] = $contactForm[2].value;
```

Comment:
This will work, but I would like you to use the `elements` property of the form element to access each value by name.
![Screen Shot 2021-02-03 at 11 53 25 AM](https://user-images.githubusercontent.com/32880187/106801293-6e410280-6616-11eb-9cfb-6164cee7f8ba.png)